lib/compiler_rt/extendhfsf2.zig

```diff
@@ -5,22 +5,17 @@ pub const panic = common.panic;
 
 comptime {
     if (common.gnu_f16_abi) {
-        @export(__gnu_h2f_ieee, .{ .name = "__gnu_h2f_ieee", .linkage = common.linkage });
+        @export(__extendhfsf2, .{ .name = "__gnu_h2f_ieee", .linkage = common.linkage });
     } else if (common.want_aeabi) {
         @export(__aeabi_h2f, .{ .name = "__aeabi_h2f", .linkage = common.linkage });
-    } else {
-        @export(__extendhfsf2, .{ .name = "__extendhfsf2", .linkage = common.linkage });
     }
```

Comment:
This exports the "standard compiler-rt" symbol names on all targets, since this is what stage2 calls for softfloat in the backend.

Let me know if this is wrong - We can make the softfloat calls to the arch-specific ABI instead of re-exporting.